Config/Global.xcconfig

```diff
@@ -4,6 +4,7 @@
 MS_WATCH_ARCHS = armv7k arm64_32
 MS_SIM_ARCHS = x86_64 i386
 SDKROOT = iphoneos
+TARGETED_DEVICE_FAMILY = 1,2
```

Comment:
This isn't a global anymore and just taking override benefits. I would suggest you to explicitly set this in iOS and macOS xcconfigs.